cookies/views.py

```diff
@@ -1136,3 +1136,46 @@ def trigger_giles_submission(request, resource_id, relation_id):
                              " connecting to the redis message passing"
                              " backend.")
             return HttpResponse(task)
+
+
+@login_required
+def bulk_action_resource(request):
+    """
+    Curator can perform actions with resources selected
```

Comment:
Can you expand on this a bit? What information is received, and what is presented to the user?